### DISCUSSION TITLE
Spark

### DISCUSSION THREAD

**Discussion by Draghonir** [PaperMC/Paper#11123] [Ideas]
### Is your feature request related to a problem?

After adding Spark under Paper, I noticed that the tab for viewing plugins is missing. Additionally, the TPS command no longer opens Spark by default.

### Describe the solution you'd like.

nee in "Is your feature request related to a problem?"

### Describe alternatives you've considered.

see in "Is your feature request related to a problem?"

**Comment by kashike** [comment]
https://github.com/PaperMC/Paper/commit/9aea24002a7b94f944d7275e7d9b0235772583b4
